src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Publish.targets

```diff
@@ -92,40 +115,4 @@
 
   </Target>
 
-  <Target Name="CopyNativeBinary" AfterTargets="Publish">
```

Comment:
Any opinion on leaving this as an empty target to make this less breaking? There are some hits on Github with dependencies on this target. In .NET 10, I think everyone can just switch to depending on Publish, so the fix is easy, but maybe tolerating a line here with a comment for a couple release would be fine too.

Example: https://github.com/microsoft/node-api-dotnet/blob/50061dd8778a1b186900a41a0487e7db5582693d/src/NodeApi/NodeApi.csproj#L37